src/applications/disability-benefits/new-686/config/chapters/stepchild-no-longer-part-of-household/stepchild-information/stepchild-information.js

```diff
@@ -1,18 +1,20 @@
 import { genericSchemas } from '../../../generic-schema';
 import { stepchildTitle } from './helpers';
+import { stepchildInfo } from '../stepchildren/helpers';
```

Comment:
This came up in one of my own pull requests. React docs suggest user defined components should be capitalized - [more here](https://reactjs.org/docs/jsx-in-depth.html#user-defined-components-must-be-capitalized)

Reply:
Good call out. Updated.